src/ServiceControl/Infrastructure/Settings/Settings.cs

```diff
@@ -58,8 +62,18 @@ public class Settings
             DataSpaceRemainingThreshold = GetDataSpaceRemainingThreshold();
             TimeToRestartErrorIngestionAfterFailure = GetTimeToRestartErrorIngestionAfterFailure();
             DisableExternalIntegrationsPublishing = SettingsReader.Read(SettingsRootNamespace, "DisableExternalIntegrationsPublishing", false);
+
+            PersistenceResolver = manifest =>
```

Comment:
Nit: Can we make this a static lambda?

Would it be worthwhile to align the transport loading with a function too?

Reply:
> Would it be worthwhile to align the transport loading with a function too?

Yes, that was the plan! First, I wanted to see what tests were failing and get those working, and then go back and clean things up.